config/build.ts

```diff
@@ -267,7 +267,8 @@ async function processSnippets(processedSnippets: Dictionary<SnippetProcessedDat
 
         let snippetOfficeReferenceIsOk =
             officeJsReferences[0] === canonicalOfficeJsReference ||
-            (group.indexOf('preview-apis') >= 0 && officeJsReferences[0] === betaOfficeJsReference);
+            (group.indexOf('preview-apis') >= 0 && officeJsReferences[0] === betaOfficeJsReference) ||
+            (group.indexOf('display-items') >= 0 && officeJsReferences[0] === betaOfficeJsReference);
```

Comment:
why this change?

Reply:
Can you move the samples to the 99-preview-apis folder?

Reply:
Yeah, undid this change by moving to th e99-preview-apis folder.